<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ledOneImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../Downloads/idea.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledOneCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledTwoCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="ledTwoImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../Downloads/idea.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledThreeCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="ledThreeImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../Downloads/idea.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledFourCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="ledFourImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../Downloads/idea.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Times</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchLedButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>LED Switching</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeInput">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="shineLedButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>LED Shining</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
